substrate/frame/fast-unstake/src/tests.rs

```diff
@@ -816,19 +816,19 @@ mod on_idle {
 			CurrentEra::<T>::put(BondingDuration::get());
 
 			// create a new validator that 100% not exposed.
-			Balances::make_free_balance_be(&42, 100 + Deposit::get());
-			assert_ok!(Staking::bond(RuntimeOrigin::signed(42), 10, RewardDestination::Staked));
-			assert_ok!(Staking::validate(RuntimeOrigin::signed(42), Default::default()));
+			Balances::make_free_balance_be(&43, 100 + Deposit::get());
```

Comment:
why? 

also could remove repetition of a magic number by assigning to a variable